src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
###AWAS2GSXg9QtoM543Hdv###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_20@9
A "NullPointerException" could be thrown; "message" is nullable here.